frontend/src/component/Sidebar/WorkspaceListItem.jsx

```diff
@@ -67,17 +57,24 @@ class WorkspaceListItem extends React.Component {
       >
         <Link
           className='sidebar__content__navigation__workspace__item__wrapper'
-          onClick={props.onClickTitle}
           to={PAGE.WORKSPACE.DASHBOARD(props.workspaceId)}
         >
           {(props.canDrop && props.isOver) && (
-            <i className={`fa ${this.getIcon()} sidebar__content__navigation__workspace__item__dragNdrop`} />
+            <i className={`fa fa-fw ${this.getIcon()} sidebar__content__navigation__workspace__item__dragNdrop`} />
+          )}
+
+          {props.level > 0 && (
+            <div style={{ marginLeft: `${(props.level - 1) * 20 + 10}px` }}>&#8735;</div>
```

Comment:
please explain the "* 20 + 10" with a comment

Reply:
Updated.